custom_components/hacs/websocket/repositories.py

```diff
@@ -207,7 +222,11 @@ async def hacs_repositories_remove(
     msg: dict[str, Any],
 ) -> None:
     """Remove custom repositoriy."""
-    hacs: HacsBase = hass.data.get(DOMAIN)
+    if (hacs := hass.data.get(DOMAIN)) is None:
+        # HACS is not properly initialized
+        connection.send_message(websocket_api.result_message(msg["id"], {}))
```

Comment:
For websockets, send errors instead of empty arrays/dictionaries if HACS is not initialized